app/views/finders/_facet_collection.html.erb

```diff
@@ -14,35 +14,26 @@
         label_text: label_text
       } %>
     </div>
-    <% if facets.select(&:filterable?).any? %>
-      <button class="app-c-button-as-link app-mobile-filters-link js-show-mobile-filters"
-        data-toggle="mobile-filters-modal" data-target="facet-wrapper"
-        data-module="gem-track-click" data-track-category="filterClicked"
-        data-track-action="mobile-filter-button" data-track-label="">
-        Filter <span class="govuk-visually-hidden"> results</span>
-        <span class="js-selected-filter-count"><%= sanitize facet_tags.display_total_selected_filters %></span>
-      </button>
-    <% end %>
   <% end %>
 
-  <div data-module="gem-track-click" data-track-category="filterClicked"
```

Comment:
There appears to be some tracking that's been lost in this update. Do we know if this is still needed?